File: tests/project_multilang/java/input/Caller.java
package demo;
public class Caller {
  void run(Object c) {
    ((ICallback) c).onReady();
    if (((ICallback) c).isEnabled()) {
      System.out.println("on");
    }
  }

  void multiline(boolean flag) {
    boolean isOneWay =
        !BuildConfig.IS_PRODUCTION
            && flag
            && other();
    use(isOneWay);
  }

  boolean other() { return true; }
  void use(boolean b) { System.out.println(b); }
}

File: tests/project_multilang/dart/input/caller.dart
void run(Object c) {
  (c as ICallback).onReady();
  if ((c as ICallback).isEnabled()) {
    print("on");
  }
}

void multiline(bool flag) {
  bool isOneWay =
      !BuildConfig.IS_PRODUCTION
          && flag
          && other();
  use(isOneWay);
}

bool other() => true;
void use(bool b) { print(b); }
